scraper.rb

```diff
@@ -6,56 +6,31 @@
 require 'pry'
 require 'scraped'
 require 'scraperwiki'
+require 'require_all'
```

Comment:
I don't think you need to require `date` here any more, as nothing in this file is using it any more. it would also be good to maintain the sort-order of the `require` statements.